scripts/pinned_build.sh

```diff
@@ -57,13 +57,42 @@ popdir() {
 }
 
 try(){
+    echo "$ $*"
     "$@"
     res=$?
     if [[ ${res} -ne 0 ]]; then
-        exit 255
+        exit $res
     fi
 }
 
+install_dependencies() {
+    echo 'Installing package dependencies.'
+    if [[ "$(uname)" == 'Linux' && -f /etc/debian_version ]]; then
+        if [[ "$(id -u)" != '0' ]]; then # if not root, use sudo for aptitude
```

Comment:
This could be apt-get or something more generic.
```suggestion
        if [[ "$(id -u)" != '0' ]]; then # if not root, use sudo for package management
```